code/modules/mob/living/living_say.dm

```diff
@@ -277,8 +277,8 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 			var/empty_indexes = get_empty_held_indexes() //How many hands the player has empty
 			if(length(empty_indexes) == 1 || !mute.get_bodypart(BODY_ZONE_L_ARM) || !mute.get_bodypart(BODY_ZONE_R_ARM))
 				message = stars(message)
-			if(length(empty_indexes) == 0)//Both hands full, can't sign
-				to_chat(src, "<span class='warning'>You can't sign with your hands full!</span.?>")
+			if(length(empty_indexes) == 0 || length(empty_indexes) == 1 && (!mute.get_bodypart(BODY_ZONE_L_ARM) || !mute.get_bodypart(BODY_ZONE_R_ARM)))//All existing hands full, can't sign
```

Comment:
```suggestion
			if(length(empty_indexes) == 0 || ( length(empty_indexes) == 1 && (!mute.get_bodypart(BODY_ZONE_L_ARM) ) || !mute.get_bodypart(BODY_ZONE_R_ARM)))//All existing hands full, can't sign
```
Okay, now here you do need parentheses for clarity, you are mixing `||` and `&&` checks.

Reply:
I'm fairly certain that adding those two parentheses would break a fair bit of the code, as the `)` you suggest is inside of a different pair of parentheses. It would make it so while the left arm being cut off would result in the correct behavior when holding items, having the right arm cut off in any capacity would simply make one unable to sign at all.

Reply:
You are right. This would be the correct way:
```suggestion
			if((length(empty_indexes) < 2 && (!mute.get_bodypart(BODY_ZONE_L_ARM) || !mute.get_bodypart(BODY_ZONE_R_ARM)))//All existing hands full, can't sign
```